pl-l2d42c8/pl-l2d42c8.xml

```diff
@@ -59,15 +59,15 @@
           est, quod interdum ex superbia non proveniat; nullum etiam quod ex cupiditate
           aliquando non descendat. Sunt enim nonnulli hominum, qui ex cupiditate fiunt superbi; et
           aliqui ex superbia fiunt cupidi. Est enim, ut ait<quote>
-
+        <ref>
           <name>Augustinus<name>
+        <ref>
```

Comment:
@cgguerra notice here that your second `<ref>` on code line 64 should be a closing `</ref>`. So it should have a slash in it, like so : `</ref>`